tiledb/sm/filesystem/posix.h

```diff
@@ -291,19 +288,8 @@ class Posix : public FilesystemBase {
   static Status write_at(
       int fd, uint64_t file_offset, const void* buffer, uint64_t buffer_size);
 
-  /**
-   * Parse config to get posix permissions for creating new files
-   * @param permissions parsed permissions are set to this parameter
-   * @return Status
-   */
-  Status get_posix_file_permissions(uint32_t* permissions) const;
-
-  /**
-   * Parse config to get posix permissions for creating new directories
-   * @param permissions parsed permissions are set to this parameter
-   * @return Status
-   */
-  Status get_posix_directory_permissions(uint32_t* permissions) const;
```

Comment:
Just leaving a note for review that these were removed since the Posix class was updated to store member variables for relevant configurations instead of the full config. The only uses of these two functions were in `posix.cc` and the new members were used in place of these helpers.

Reply:
It's a good change.